Functions.Templates/Templates-v2/BlobTrigger-JavaScript/template.json

```diff
@@ -0,0 +1,47 @@
+{
+    "author": "Eric Jizba",
+    "name": "Azure Blob Storage trigger",
+    "description": "$BlobTrigger_description",
+    "programmingModel": "v4",
+    "language": "JavaScript",
+    "jobs": [
+        {
+            "name": "Create New Function",
+            "type": "CreateNewFile",
+            "inputs": [
+                {
+                    "assignTo": "$(FUNCTION_NAME_INPUT)",
+                    "paramId": "functionName",
+                    "defaultValue": "storageBlobTrigger"
+                },
+                {
+                    "assignTo": "$(CONNECTION_INPUT)",
+                    "paramId": "connection"
+                },
+                {
+                    "assignTo": "$(PATH_INPUT)",
+                    "paramId": "path"
+                }
+            ],
+            "actions": [
+                "readFile",
+                "writeFile"
+            ]
+        }
+    ],
+    "actions": [
+        {
+            "name": "readFile",

```

Comment:
Nit: I would recommend a different name for this. I think "readFile" implied to me that we were reading the _customer's_ file and then writing to it, or something like that. I'd suggest something like "readTemplate" or "getTemplateContents" or something like that

Reply:
updated